cli/azd/pkg/templates/template_manager.go

```diff
@@ -177,8 +178,22 @@ func PromptTemplate(
 	for _, template := range templates {
 		templateChoice := template.Name
 		if isInteractive {
-			repoPath := output.WithGrayFormat("(%s)", template.RepositoryPath)
-			templateChoice += fmt.Sprintf("\n  %s\n", repoPath)
+			if termlink.SupportsHyperlinks() {
+				trimmedPath := strings.Split(
+					strings.TrimRight(
+						strings.TrimLeft(
+							template.RepositoryPath,
+							"\x1b]8;;https://github.com/"),
+						"\x1b]8;;\x07\u001b[0m"),
+					"\x07")
```

Comment:
Same issue with termlink the terminal crashed when I tried this code:
```
url, err := Absolute(template.RepositoryPath)
if err == nil {
  templateChoice += fmt.Sprintf("\n  (%s)\n", output.WithHyperlink(url, template.RepositoryPath))
} else {
  templateChoice += fmt.Sprintf("\n  (%s)\n", template.RepositoryPath)
}
```
![image](https://github.com/Azure/azure-dev/assets/64026625/5167d9d3-7c4d-4111-b17b-2bd73cd781c0)

Reply:
I didn't know that we had a hyperlink function I will use it instead of termlink.

Reply:
Hi @john0isaac what terminal is it crashing on? We do have existing usage of `WithHyperlink`. It is possible that this is unreported, but it'd be good to know.

Did you fix the crash in the original code by checking `termlink.SupportsHyperlinks`?

Reply:
Powershell 
Let me check git bash also.
it only crashes on the init command not the templates list and I think this maybe because of the moving UI text feature.

Reply:
Gitbash also crashes..
![image](https://github.com/Azure/azure-dev/assets/64026625/b17e1535-2cd3-4028-ba21-0c5f9bff161e)

Reply:
I fixed the crashing by un implementing the feature 😆 
when you run init I returned the old username/reponame not a clickable link but for the template list it was clickable.

Reply:
> when you run init I returned the old username/reponame not a clickable link but for the template list it was clickable

I see. Will need to dig in further.